observablehq.config.ts

```diff
@@ -1,3 +1,7 @@
+import {formatPrefix} from "d3-format";
+
+const {stargazers_count} = await github("/repos/observablehq/framework").catch(() => ({stargazers_count: 4242}));
```

Comment:
nit: I'd default to null or undefined if the github API call fails. This might require an additional test below to avoid showing 0.0k/NaNk stars (although it could be ok too to show NaN if we can't get the information).

Reply:
I was only thinking of local development. In CI I’d want the deploy to fail if we’re not able to talk to GitHub, so that we always show the correct star count. I’ve changed the code to rethrow in CI.